runtime/constants.go

```diff
@@ -137,3 +137,8 @@ var knownStatusCodes = map[int]bool{
 	http.StatusNotExtended:                   true, // 510
 	http.StatusNetworkAuthenticationRequired: true, // 511
 }
+
+var noContentStatusCodes = map[int]bool{
```

Comment:
noBodyStatusCodes or something similar would be better since `noContentStatusCode` implies only 204.

Reply:
I think, Body and Content sound similar, and noBodyStatusCode**s** plural may signify that we are including all no content ones, and avoid confusion.